ibridges/data_operations.py

```diff
@@ -35,6 +35,7 @@ def upload(
     options: Optional[dict] = None,
     dry_run: bool = False,
     metadata: Union[None, str, Path] = None,
+    **kwargs
```

Comment:
Good idea! I was already worried about the plethora of options for which we made single variables.